docs/user/getting-started.md

```diff
@@ -199,52 +199,51 @@ Finally, move it into a location within your shell's search path.
 Retrieve the password for the "demouser" account that's been automatically generated for you during installation:
 
 ```console
-$ kubectl get secret/keycloak-demouser-secret -n flightctl -o=jsonpath='{.data.password}' | base64 -d
-
-[...]
+password=$(kubectl get secret/keycloak-demouser-secret -n flightctl -o=jsonpath='{.data.password}' | base64 -d)
 ```
 
-Use the CLI to log into the Flight Control Service:
+Headless login:
 
 ```console
-$ flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
-
-[...]
+flightctl login <flightctl_api_url> -u demouser -p ${password}
 ```
 
-In the web browser that opens, use the login "demouser" and the password you retrieved in the previous step.
+> 📌 For headless login to work, the OIDC provider of your choice needs to have Direct Access grant enabled
 
-Verify you can now access the service via the CLI:
+Login using web browser:
 
 ```console
-$ flightctl get devices
-
-NAME                                                  OWNER   SYSTEM  UPDATED     APPLICATIONS  LAST SEEN
+flightctl login <flightctl_api_url> --web
 ```
 
 ### ACM deployment
 
-Use the CLI to log into the Flight Control Service:
+You will be logging in with ACM login credentials.
 
-```console
-$ flightctl login https://api.flightctl.127.0.0.1.nip.io/ --web --insecure-skip-tls-verify
+Headless login:
 
-[...]
+```console
+flightctl login <flightctl_api_url> -u <acm_user> -p <password>
```

Comment:
How can the user figure out what that URL is?

Reply:
For ACM deployment, it will always be through looking at routes, but the Standalone deployment is tricky due to multiple options (Route, Gateway or Node port). After `helm install`, we actually print out API URL so I've added this info to the doc.